lib/compiler.js

```diff
@@ -95,6 +96,22 @@ Compiler.prototype.compile = function() {
     `;
   }
 
+  if (this.hasAttrsProp) {
+    js = `
+      var __vjadeStringifyAttrsIfObj = function(attrs) {
+        if (typeof attrs !== 'object') return attrs;
+        const copy = Object.assign({}, attrs);
+        for (const k in attrs) {
```

Comment:
```suggestion
        for (const [k, v] of Object.entries(attrs)) {
```
or
```suggestion
        for (const k of Object.keys(attrs)) {
```
but don't do a raw `k in attrs` without all the hasOwnProperty shenanigans